.github/workflows/test.yml

```diff
@@ -3,19 +3,20 @@ on: [pull_request]
 jobs:
   build:
     name: Test
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
         node-version: [10.x, 12.x, 13.x]
     steps:
       - uses: actions/checkout@v1
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - name: Install dependences
+      - name: Install dependencies
         run: |
-          npm install -g yarn 
+          npm install -g yarn
           sudo apt-get install tree
```

Comment:
My bad, I forget this line...

Reply:
Fixed: https://github.com/vinissimus/next-translate/pull/103/commits/5551a763c570ed8641787f8ccdd7a042d3089879